### PR TITLE
Attempt to access multimedia blob when finding missing media

### PR DESCRIPTION
**Pull request by proteusvacuum**
## Product Description

With remote hosted CommCare we are often finding that multimedia blobs are missing for some reason. When this happens, just checking that the multimedia map is present doesn't re-pull images. Adding this check pulls the missing file if the blob is missing for some reason

## Feature Flag

Remote linked domains

### Safety story

Tested on staging. Only relevant for downstream linked domains. 

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by proteusvacuum** [comment]
FYI @sravfeyn , not sure if this is useful for eCHIS who were having similar issues I think.
This should fix errors that look like this when downloading an app:
```
CommCareImage fa7804b0f95d3cda826406ab65642c37 attachment: 'f9286057826172d49d28ad256398c526-1200x1200.PNG'"
```

**Review by gherceg — APPROVED** [review]

File: corehq/apps/linked_domain/util.py [hunk 1]
```diff
@@ -96,6 +96,7 @@ def _get_missing_multimedia(app, old_multimedia_ids=None):
             continue
         try:
             local_media = CommCareMultimedia.get(media_info['multimedia_id'])
+            local_media.get_display_file()
```
Comment (mkangia) [inline corehq/apps/linked_domain/util.py:99]: Is this going to slow down the pull since this would load files for each MM?

Comment (proteusvacuum) [reply corehq/apps/linked_domain/util.py:99]: Yes, it will probably take longer.

Comment (sravfeyn) [reply corehq/apps/linked_domain/util.py:99]: Should this happen only when the media is missing?

**Comment by sravfeyn** [comment]
I think eCHIS is missing media in the DB itself, so I am not sure if this would solve their issue.

File: corehq/apps/linked_domain/util.py (hunk 1 above)
Comment (proteusvacuum) [reply corehq/apps/linked_domain/util.py:99]: @sravfeyn , can you elaborate? Multimedia is "stored" in a few places - CommCareMultimedia stores media metadata in couch with a pointer to the actual file in the blobdb.  Line 98 checks if it is missing from couch, and I added line 99 to make sure the actual file exists in the blobdb. So this is basically checking that the media exists.

**Comment by mkangia** [comment]
@proteusvacuum 
Do we want to add some logging to notify when this actually happens to shoot down on why this is happening?

There was a similar situation on ICDS earlier where I believe the build/ccz creation was made to fail if multimedia was missing to avoid it failing later on phone. Sharing related PRs if you want to take a look.
https://github.com/dimagi/commcare-hq/commit/6a726cc6c20423eb9d009cc9471860e148cfb6b3#diff-d840cf1f750aceea9aa25a851af473652f1c88963287f319badf69bbbd6a01bb
https://github.com/dimagi/commcare-hq/pull/23287

**Pull request merged**